### PR TITLE
[Insight] Add tx replacement links; Testnet banner; Minor tweaks

### PR DESCRIPTION
**Pull request by kajoseph**
This PR:

* Adds a link to the replacement tx when a tx has been RBF'd or replaced by nonce
* Adds a banner when on testnet
* Adds some useful fields to ETH and UTXO tx page
* Minor copy tweaks for accuracy

### CODE REVIEW

**Review by escottalexander — COMMENTED** [review]
Nice changes. Left a couple of comments but otherwise this looks great.

File: packages/insight/src/pages/transaction.tsx [hunk 1]
```diff
@@ -122,10 +130,14 @@ const TransactionHash: React.FC = () => {
...
+                <Info
+                  message={`This transaction was replaced by ${transaction.replacedByTxid}`}
+                  type={'error'}
+                  onClick={() => goToTx(transaction.replacedByTxid)}
+                /> :
                 <Info
-                  message={
-                    'This transaction is invalid and will never confirm, because some of its inputs are already spent.'
-                  }
+                  message={`This transaction was replaced by another transaction that ${transaction.chain === 'ETH' ? 'used the same nonce' : 'spent some of it\'s inputs'}.`}
```
[the remark this answers is not in the document]
Comment (kajoseph) [reply packages/insight/src/pages/transaction.tsx:140]: It does have a click event for ETH _if_  `transaction.replacedByTxid` exists. This message is just the fallback for if `transaction.replacedByTxid` doesn't exist (e.g. an tx from before adding that field).

**Review by escottalexander — APPROVED** [review]
Looks good!

**Review by lanchana — APPROVED** [review]
LGTM!

**Review by nitsujlangston — APPROVED** [review]

**Pull request merged**